app/views/unattended/provisioning_templates/snippet/preseed_kernel_options.erb

```diff
@@ -27,16 +37,25 @@ description: options for the kernel / preseed startup initialization
   if @host.provision_interface.vlanid.present?
     options << "netcfg/use_vlan=true netcfg/vlan_id=#{@host.provision_interface.vlanid}"
   end
-  options << "locale=#{host_param('lang') || 'en_US'}"
```

Comment:
why are you dropping this line?

Reply:
Sorry for the late reply - I am dropping it because this line is a duplicate. The same line is already in line 65.